app/core/components/Search.js

```diff
@@ -37,6 +37,9 @@ export function Search({className = ""}) {
     },
     [setIsOpen, setInitialQuery],
   )
+  const ConvertToHtmlTag = (value) => {
+    return value.replace(/&amp;/g, "&").replace(/&lt;/g, "<").replace(/&gt;/g, ">")
+  }
```

Comment:
Let's keep the function name lowercase
```suggestion

  const convertToHtmlTag = (value) => {
    return value.replace(/&amp;/g, "&").replace(/&lt;/g, "<").replace(/&gt;/g, ">")
  }
```